### PR TITLE
Fix pyright

### PR DESCRIPTION
**Pull request by crsegerie**

### CODE REVIEW

File: kili/types.py [hunk 1]
```diff
@@ -6,24 +6,29 @@
...
 from dataclasses import dataclass
 
+from typing_extensions import TypedDict
 
-@dataclass
-class License:
+from kili.enums import InputType, LicenseType
+
+
+class License(TypedDict):
```
Comment (PierreLeveau) [inline kili/types.py:14]: separate the replacement of dataclasses by the typedicts

File: requirements.txt [hunk 2]
```diff
@@ -8,3 +8,4 @@ tqdm
 typeguard
```
Comment (PierreLeveau) [inline requirements.txt:8]: ☝️ rm [recipes/download_google_drive](https://github.com/kili-technology/kili-python-sdk/pull/657/files#diff-aceaba362149f189bc98fbfe0646e9142bf339484a4c506985ac00e1b1de6dd9)

File: requirements.txt [hunk 3]
```diff
@@ -8,3 +8,4 @@ tqdm
...
 websocket-client
 typing_extensions
+google-cloud
```
Comment (PierreLeveau) [inline requirements.txt:11]: rm from baseline requirements

File: test/services/export/fakes/fake_kili.py [hunk 4]
```diff
@@ -2,12 +2,11 @@
 Fake Kili object
 """
 
-from test.services.export.fakes.fake_data import asset_image
+from test.services.export.fakes.fake_data import asset_image  # type:ignore
```
Comment (PierreLeveau) [inline test/services/export/fakes/fake_kili.py:5]: rm # type:ignore

File: test/test_client_project.py [hunk 5]
```diff
@@ -33,13 +34,13 @@ def test_get_unexisting_project(self, mocker):
         mocker.patch("kili.client.Kili.projects", side_effect=mocked__projects)
         kili = Kili()
         with pytest.raises(NotFound):
-            kili.get_project("this_project_id_do_not_exist")
+            kili.get_project(ProjectId("this_project_id_do_not_exist"))
```
Comment (PierreLeveau) [inline test/test_client_project.py:37]: rm need to transtype

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: kili/client.py [hunk 6]
```diff
@@ -27,6 +27,7 @@
 from kili.queries.project_user import QueriesProjectUser
 from kili.queries.project_version import QueriesProjectVersion
 from kili.queries.user import QueriesUser
+from kili.services.export.typing import ProjectId
```
[the remark this answers is not in the document]
Comment (crsegerie) [reply kili/client.py:30]: in fact, It does not make sense to make this change considering the content of typing.py

**Review by PierreLeveau — APPROVED** [review]
LGTM, but wait until the release is done before merge

**Pull request merged**